docs/openhabian.md

```diff
@@ -77,7 +77,7 @@ Learn more about the Raspberry Pi as your platform for openHAB and about the req
 
 **Setup:**
 
-- [Download the latest "openHABianPi" SD card image file](https://github.com/openhab/openhabian/releases) (Note: the file is *xz* compressed)
+- [Download the latest "openHABian" SD card image file](https://github.com/openhab/openhabian/releases) (Note: the file is *xz* compressed)
 - Write the image to your SD card (e.g. with [Etcher](https://etcher.io), able to directly work with *xz* files)
```

Comment:
@Confectrician IIRC you auto-copy docs/ document to docs.openhab.org, don't you ?
How to proceed with these images?
There's also a new Debug guide we should point at docs/openhabian-DEBUG.md

Reply:
Can you also change includes/bash_profile to point at the debug guide (see end of functions/menu.bash) rather than the current URL and make a new pic of that ?